probe-rs-cli-util/src/rtt.rs

```diff
@@ -412,14 +429,36 @@ impl RttActiveTarget {
     }
 
     /// Polls the RTT target on all channels and returns available data.
+    /// Errors on any channel will be ignored and the data (even if incomplete) from the other channels will be returned.
+    // NOTE: Keeping this signature and "ignore errors" behaviour for backwards compatibility.
     pub fn poll_rtt(&mut self, core: &mut Core) -> HashMap<String, String> {
```

Comment:
Can we add a deprecation attribute? https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute